tests/api/transactions/update-state.test.js

```diff
@@ -39,4 +44,120 @@ describe('transactions:update-state', () => {
       err.body.status_verbose.should.equal('wrong user')
     })
   })
+
+  describe('side effects', () => {
+    describe('one way transactions', () => {
+      const itemData = { transaction: _.sample([ 'giving', 'selling' ]), listing: 'public' }
+
+      it('should set item.busy=false when the transaction is just requested', async () => {
```

Comment:
i know its backshedding, but its my first comments :)
set is an action, while tests are testing states
```suggestion
      it('should be false when the transaction is requested', async () => {
```

Reply:
backshedding on your backshedding: it's bikeshedding ;p

Reply:
done in 24e03ab